keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -65,7 +65,7 @@
             }
           },
           "default": {
-            "description": "Default response. It will be deserialized as per the error definition specified in the schema. An exception will be thrown.",
+            "description": "Default response. It will be deserialized per the error definition specified in the schema. An exception will be thrown.",
```

Comment:
Thanks Bryan, that seems a good way to go. I'll use it. 
"CDN error response describing why the operation failed."